sdc/tests/tests_perf/generator.py

```diff
@@ -16,8 +16,7 @@ def gen(cases, method, class_add, typ, prefix=''):
         name = test_params.name
         if test_params.params:
             name += "_" + test_params.params
-        func_name = 'test_{}_{}_{}'.format(typ, prefix, name)
-        func_name = to_varname(func_name).replace('__', '_')
+        func_name = to_varname('test_{}_{}_{}'.format(typ, prefix, name))
```

Comment:
Please apply `to_varname` only for `test_params.params` and use `'_'.join(...)` instead of formatted string.